dev-environments/plain-http-upstream/Makefile

```diff
@@ -6,7 +6,7 @@ WORKDIR := $(patsubst %/,%,$(dir $(MKFILE_PATH)))
 DOCKER ?= $(shell which docker 2> /dev/null || echo "docker")
 
 gateway: ## run gateway configured to access plain HTTP 1.1 upstream
-	$(DOCKER) compose -f docker-compose.yml up --attach gateway
```

Comment:
Sorry about this. This was not my intention. This happened because we merged https://github.com/3scale/APIcast/pull/1442 before #1441 and the rebase tricked me. Actually it is my fault for not reviewing the changes after rebase and (too) quick merging.

I will create a new PR to fix this

cc @tkan145

Reply:
The revert of the revert https://github.com/3scale/APIcast/pull/1444